### PR TITLE
Fix: Replace dates in the past with dates in the future #15

### PR DESCRIPTION
**Pull request by biswain2309**
Replaced all the past dates in all the SDK's with future dates. Please review.

### CODE REVIEW

**Comment by tsolakoua** [comment]
Thank you so much, @biswain2309 for your contribution! We will review your PR from next week! Happy Hacktoberfest :)

File: flight_busiest_traveling_period/v1/get/Java SDK/FlightBusiestPeriod.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public static void main(String[] args) throws ResponseException {
     // Flight Busiest Traveling Period
     Period[] busiestPeriods = amadeus.travel.analytics.airTraffic.busiestPeriod.get(Params
       .with("cityCode", "MAD")
-      .and("period", "2017")
```
Comment (tsolakoua) [inline flight_busiest_traveling_period/v1/get/Java SDK/FlightBusiestPeriod.java:19]: This should not change, as the API takes as input dates in the past.

File: flight_busiest_traveling_period/v1/get/Node SDK/flight_busiest_traveling_period.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ var amadeus = new Amadeus({
 // What were the busiest months for Madrid in 2017?
 amadeus.travel.analytics.airTraffic.busiestPeriod.get({
   cityCode: 'MAD',
-  period: '2017',
```
Comment (tsolakoua) [inline flight_busiest_traveling_period/v1/get/Node SDK/flight_busiest_traveling_period.js:10]: This endpoint should not change, as the API takes as input dates in the past.

File: flight_busiest_traveling_period/v1/get/Python SDK/flight_busiest_traveling_period.py [hunk 3]
```diff
@@ -8,9 +8,10 @@
 
 try:
     '''
-    What were the busiest months for Madrid in 2017?
+    What were the busiest months for Madrid in 2022?
     '''
-    response = amadeus.travel.analytics.air_traffic.busiest_period.get(cityCode='MAD', period='2017', direction='ARRIVING')
+    response = amadeus.travel.analytics.air_traffic.busiest_period.get(
```
Comment (tsolakoua) [inline flight_busiest_traveling_period/v1/get/Python SDK/flight_busiest_traveling_period.py:13]: This endpoint should not change, as the API takes as input dates in the past.

File: flight_busiest_traveling_period/v1/get/curl/flight_busiest_traveling_period.sh [hunk 4]
```diff
@@ -5,4 +5,4 @@ https://test.api.amadeus.com/v1/security/oauth2/token \
 -d "grant_type=client_credentials&client_id=$AMADEUS_CLIENT_ID&client_secret=$AMADEUS_CLIENT_SECRET" \
 | grep access_token | sed 's/"access_token": "\(.*\)"\,/\1/' | tr -d '[:space:]')
 
-curl -X GET "https://test.api.amadeus.com/v1/travel/analytics/air-traffic/traveled?originCityCode=MAD&period=2017-01&direction=ARRIVING" -H "Authorization: Bearer $ACCESS_TOKEN" -k
```
Comment (tsolakoua) [inline flight_busiest_traveling_period/v1/get/curl/flight_busiest_traveling_period.sh:8]: This endpoint should not change, as the API takes as input dates in the past.

File: flight_busiest_traveling_period/v1/get/Android SDK/FlightBusiestPeriod.kt [hunk 5]
```diff
@@ -19,7 +19,7 @@ val amadeus = Amadeus.Builder(context)
 scope.launch {
   when (val busiestPeriods = amadeus.travel.analytics.airTraffic.busiestPeriod.get(
                       cityCode = "MAD",
-                      period = "2017",
+                      period = "2022",
```
[the remark this answers is not in the document]
Comment (biswain2309) [reply flight_busiest_traveling_period/v1/get/Android SDK/FlightBusiestPeriod.kt]: Resolved the issue.

File: flight_busiest_traveling_period/v1/get/Java SDK/FlightBusiestPeriod.java (hunk 1 above)
Comment (biswain2309) [reply flight_busiest_traveling_period/v1/get/Java SDK/FlightBusiestPeriod.java]: Resolved the issue.

File: flight_busiest_traveling_period/v1/get/Node SDK/flight_busiest_traveling_period.js (hunk 2 above)
Comment (biswain2309) [reply flight_busiest_traveling_period/v1/get/Node SDK/flight_busiest_traveling_period.js]: Resolved the issue.

File: flight_busiest_traveling_period/v1/get/Python SDK/flight_busiest_traveling_period.py (hunk 3 above)
Comment (biswain2309) [reply flight_busiest_traveling_period/v1/get/Python SDK/flight_busiest_traveling_period.py:13]: Resolved the issue.

File: flight_busiest_traveling_period/v1/get/iOS SDK/FlightBusiestPeriod.swift [hunk 6]
```diff
@@ -7,7 +7,7 @@ var amadeus: Amadeus = Amadeus(
 
 // Flight Busiest Traveling Period
 amadeus.travel.analytics.airTraffic.busiestPeriod.get(params: ["cityCode": "MAD",
-    "period": "2017",
```
[the remark this answers is not in the document]
Comment (biswain2309) [reply flight_busiest_traveling_period/v1/get/iOS SDK/FlightBusiestPeriod.swift]: Resolved the issue.

**Review by tsolakoua — APPROVED** [review]
Thank you!

**Pull request merged**